src/native/libs/System.Security.Cryptography.Native.Apple/entrypoints.c

```diff
@@ -19,17 +18,25 @@
 #include "pal_ssl.h"
 #include "pal_swiftbindings.h"
 #include "pal_symmetric.h"
-#include "pal_trust_macos.h"
 #include "pal_x509.h"
-#include "pal_x509_macos.h"
 #include "pal_x509chain.h"
+#if defined(TARGET_OSX)
+#include "pal_trust_macos.h"
+#include "pal_keychain_macos.h"
+#include "pal_x509_macos.h"
+#else
+#include "pal_keychain_ios.h"
+#include "pal_x509_ios.h"
+#endif
 
 static const Entry s_cryptoAppleNative[] =
 {
+#ifdef TARGET_OSX
     DllImportEntry(AppleCryptoNative_AesGcmEncrypt)
```

Comment:
nit: maybe we can group these together and put them under a same ifdef?

Reply:
I guess so... I originally thought they had to be sorted but then I found out they are not sorted in System.Native.